examples/less-antdesign/static.config.js

```diff
@@ -0,0 +1,263 @@
+                      'not ie < 9', // React doesn't support IE8 anyway
+                    ],
+                    flexbox: 'no-2009',
+                  }),
+                ],
+              },
+            },
+            {
+              loader: 'less-loader',
+              options: {
+                sourceMap: true,
+                modifyVars: themeVariables,
+              },
+            },
+          ],
+        }
+    } else {
+      // Extract to style.css
+      lessLoader =
+        {
+          test: /\.less$/,
+          loader: ExtractTextPlugin.extract({
+            fallback: {
+              loader: 'style-loader',
+              options: {
+                hmr: false,
+                sourceMap: false,
+              },
+            },
+            use: [
+              {
+                loader: 'css-loader',
+                options: {
+                  importLoaders: 1,
+                  minimize: true,
+                  sourceMap: false,
+                },
+              },
+              {
+                loader: 'postcss-loader',
+                options: {
+                  // Necessary for external CSS imports to work
+                  // https://github.com/facebookincubator/create-react-app/issues/2677
+                  ident: 'postcss',
+                  plugins: () => [
+                    postcssFlexbugsFixes,
+                    autoprefixer({
+                      browsers: [
+                        '>1%',
+                        'last 4 versions',
+                        'Firefox ESR',
+                        'not ie < 9', // React doesn't support IE8 anyway
+                      ],
+                      flexbox: 'no-2009',
+                    }),
+                  ],
+                },
+              },
+              {
+                loader: 'less-loader',
+                options: {
+                  sourceMap: false,
+                  modifyVars: themeVariables,
+                },
+              },
+            ],
+          }),
+        }
+    }
+
+    /*
+    * Add new Loaders to default Loaders
+    * */
+
+    config.module.rules = [
+      {
+        oneOf: [
+          jsTsLoader,
+          lessLoader,
+          defaultLoaders.cssLoader,
+          defaultLoaders.fileLoader,
+        ],
+      },
+    ]
+
+    // Update ExtractTextPlugin with current instance
+    config.plugins[2] =
+      new ExtractTextPlugin({
+        filename: getPath => {
+          process.env.extractedCSSpath = 'styles.css'
+          return getPath('styles.css')
+        },
+        allChunks: true,
+      })
+
+
+    /*
+    * In the future, this segment will change or be unnecessary.
+    * Whitelisting Extensions from pre-parser and universal url-loader
+    * */
+
+    // Tell pre-processor (babel) to no-op styles since, though they are
+    // imported as modules, are not parseable code.
+    require('babel-register')
+    require.extensions['.less'] = () => {}
```

Comment:
@tannerlinsley Should be ready now! However, for the time being, we don't get around this one.